spectractor/extractor/extractor.py

```diff
@@ -238,12 +239,14 @@ def set_y_c(self):
                 else:
                     self.p[k] = 0
 
-    def set_mask(self, params=None):
+    def set_mask(self, params=None, fwhmx_clip=3*parameters.PSF_FWHM_CLIP, fwhmy_clip=parameters.PSF_FWHM_CLIP):
```

Comment:
Having complicated defaults in function definitions is often quite dangerous, but I think we should save that for a later cleanup, as that's done in quite a lot of places already.